src/testUtils/ChaiAddons.js

```diff
@@ -0,0 +1,61 @@
+export default function (chai, utils) {
+  var Assertion = chai.Assertion;
+
+  //uses Chai flags to hold on to the done callback
+  function flagDoneCallback(done) {
+    utils.flag(this, 'done', done)
+  }
+
+  Assertion.addChainableMethod('eventually', flagDoneCallback)
+
+  Assertion.addMethod('informRegisteredStore', function(store) {
+    var action = this._obj;
+    var mockPayload = {foo: 'bar'}
+    var callbackAlreadyFired = false
+    var done = utils.flag(this, 'done')
+    var ctx = this
```

Comment:
@BJK I know `ctx` is a common abbreviation, but so is `txt`, and that bit me in the ass on a card a few sprints back. How do you feel about using `context` in place of `ctx`?